.travis.yml

```diff
@@ -56,3 +56,12 @@ matrix:
       env: SCALIKEJDBC_DATABASE="hsqldb"
     - scala: 2.13.0-M4
       env: SCALIKEJDBC_DATABASE="mysql"
+    - scala: 2.13.0-M5
+      env: SCALIKEJDBC_DATABASE="mysql"
+      script:
+      - sbt
+        "++ ${TRAVIS_SCALA_VERSION}! -v"
+        config/compile
```

Comment:
I've checked the projects except for mapper-generator and test. Thus, we can add `joda-time/compile` here.
```
sbt:scalikejdbc> projects
[info] In file:/Users/kazuhiro.sera/github/oss/scalikejdbc/
[info] 	   config
[info] 	   core
[info] 	   interpolation
[info] 	   interpolation-macro
[info] 	   joda-time
[info] 	   library
[info] 	   mapper-generator
[info] 	   mapper-generator-core
[info] 	   root211
[info] 	 * scalikejdbc
[info] 	   streams
[info] 	   syntax-support-macro
[info] 	   test
```

Reply:
done